src/trex/quality_control.py

```diff
@@ -0,0 +1,378 @@
+from itertools import combinations
```

Comment:
Imports should be in this order:
1. Python standard library (`re`, `pathlib` etc)
2. External libraries (`pandas`, `seaborn`)
3. Modules within the same package (`from .utils import ...`, you don’t have that here)

There should be a newline between each section.

You could let [isort](https://pycqa.github.io/isort/) do this automatically for you (use `isort --profile=black <filename>`)

Reply:
I updated my pre-commit to have black and isort. Should we have a pre-commit config in root that we share?

Reply:
Yes, but this needs to be coordinated a bit because we should reformat the code base at the same time and add a test to CI that enforces formatting.